presto-tests/src/main/java/com/facebook/presto/tests/AbstractTestQueries.java

```diff
@@ -1640,6 +1640,155 @@ public void testRollupOverUnion()
     }
 
     @Test
+    public void testGrouping()
+            throws Exception
+    {
+        assertQuery("" +
```

Comment:
I think I know what you mean. I changed the indentation in faecced56758c196d105fc64c009ce0eda2d4381. Let me know if you disagree.